geode-lucene/src/test/java/org/apache/geode/cache/lucene/LuceneIndexMaintenanceIntegrationTest.java

```diff
@@ -198,7 +200,7 @@ public void statsAreUpdatedAfterACommit() throws Exception {
 
     FileSystemStats fileSystemStats = index.getFileSystemStats();
     LuceneIndexStats indexStats = index.getIndexStats();
-    await(() -> assertEquals(4, indexStats.getDocuments()));
+    await(() -> assertEquals(true, 4 <= indexStats.getDocuments()));
```

Comment:
Shouldn't creating a Lucene index before region creation result in a predictible stat count like the test below?